docs/docs/references/dimensions.mdx

```diff
@@ -212,3 +212,34 @@ Filters can be used to make small transformations of your values:
 | append     | Append a string to another                                             | `${value.formatted \ | append: ".html"}`|
 
 There are [many more filters available in the Liquid documentation](https://liquidjs.com/filters/overview.html).
+
+
+### Using special characters on your column names
```

Comment:
```suggestion
### Using special characters or capital letters in your column names
```

I'd suggest adding capital letters explicitly here - I wouldn't think of them as special characters as a non-engineering person 😅 but this might be totally weird wording.